generic_tracers/generic_COBALT.F90

```diff
@@ -4313,9 +4321,40 @@ subroutine generic_COBALT_update_from_source(tracer_list,Temp,Salt,rho_dzt,dzt,h
 
           ! Iron from sediment (Dale, 2015).  The maximum release from the sediment is set by ffe_sed_max.  The
           ! hyperbolic tangent requires the flux of carbon to the sediments (as mmoles m-2 day-1) in the numerator
-          ! and the bottom water oxygen concentration (in microMolar units) in the denominator:
+          ! and the bottom water oxygen concentration (in microMolar units) in the denominator. Note that ffe_sed_max
+          ! was converted to moles Fe m-2 sec-1 during parameter input, so ffe_sed has is in moles Fe m-2 sec-1
           cobalt%ffe_sed(i,j) = cobalt%ffe_sed_max * tanh( (cobalt%fntot_btm(i,j)*cobalt%c_2_n*sperd*1.0e3)/ &
                                 max(cobalt%btm_o2(i,j)*1.0e6,epsln) )
+
+          ! Additional coastal iron (Optional, default fe_coast = 0)
+          !
+          ! Coarse resolution models and/or intermediate resolution models in areas with exceptionally steep bathymetry
+          ! can under-represent coastal iron because they don't resolve shallow regions. An option to add iron through
+          ! the vertical face of the land mass has thus been included.  The flux is posed as a fraction (fe_coast) of
+          ! the sediment Fe flux (moles Fe m-2 sec-1) that would have result from the sinking organic matter flux and
```

Comment:
change "result" to "resulted" or remove "have" in this sentence "the sediment Fe flux (moles Fe m-2 sec-1) that would have result from the sinking organic matter flux and O2 level of the adjacent waters."

Reply:
Got it.  Thanks.